src/apis/auth/sign-in-google.ts

```diff
@@ -0,0 +1,60 @@
+import Elysia, { t } from "elysia";
+
+import { accessJwt, renewJwt } from "@root/plugins/jwt.plugin";
+import { UserRepository } from "@root/repositories/user.repository";
+import { getGoogleUserInfo } from "@root/services/http/get-google-user-info.ts";
+import { ENDPOINT } from "@root/shared/constant";
+
+type UserData = {
+  sub: string; // google id
+  email: string;
+};
+
+export const signInGoogle = new Elysia({
+  name: "Handler.SignInGoogle"
+})
+  .use(accessJwt)
+  .use(renewJwt)
+  .post(
+    ENDPOINT.AUTH.SIGN_IN_GOOGLE,
+    async ({ body, access, renew }) => {
+      let accessToken = "";
+      let refreshToken = "";
```

Comment:
Please limit the use of reassignment. Define tokens after we already have user and return them.